routeoptimization/snippets/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright 2022 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.example.routeoptimization</groupId>
+  <artifactId>routeoptimization-samples</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <!--
+   The parent pom defines common style checks and testing strategies for our samples.
+   Removing or replacing it should not affect the execution of the samples in anyway.
+  -->
+  <parent>
+    <artifactId>shared-configuration</artifactId>
+    <groupId>com.google.cloud.samples</groupId>
+    <version>1.2.0</version>
+  </parent>
+
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <artifactId>google-maps-routeoptimization-bom</artifactId>
+        <groupId>com.google.maps</groupId>
+        <scope>import</scope>
+        <type>pom</type>
+        <version>0.1.0</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.maps</groupId>
+      <artifactId>google-maps-routeoptimization</artifactId>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
(NB: I'm not a java packaging expert)

AIUI this isn't a duplicate dependency, the `dependencyManagement` declaration is equivalent to `libraries-bom` declared used elsewhere (see http://shortn/_72sIg41fuQ on why libraries-bom isn't used here).

Reply:
Resolving since I believe I'm not deviating from precedent modulo the differences/rationale described in the linked document.

Reply:
The linked document is outdated (2022). It also does not reference `google-maps-routeoptimization-bom`. As such it cannot be used as a justification.
Unless you can clearly explain why you need both, please remove this dependency or the BOM dependency above. If you decide to remove BOM, please update this dependency by removing `scope` (defining `compile` scope is redundant since it is a default scope) and add the version of the dependency that should be used.

Reply:
Removed BOM.